src/osx/cocoa/utils_base.mm

```diff
@@ -191,3 +191,95 @@ wxString wxGetOsDescription()
     return true;
 }
 #endif // wxUSE_DATETIME
+
+bool wxCocoaLaunch(const char* const* argv, pid_t &pid)
+{
+    // Obtains the number of arguments for determining the size of
+    // the CFArray used to hold them
+    NSUInteger cfiCount = 0;
```

Comment:
This is not used except for checking if it's 0, which could be done with just `if ( !argv )`

Reply:
I'll do that